File: thingsalarm2/src/main/java/com/example/thingsalarm/FireIDService.java
package com.example.thingsalarm;

import android.util.Log;

import com.google.firebase.iid.FirebaseInstanceId;
import com.google.firebase.iid.FirebaseInstanceIdService;



public class FireIDService extends FirebaseInstanceIdService {

    private static final String TAG = "MyFirebaseIIDService";

    @Override
    public void onTokenRefresh () {
        String tkn = FirebaseInstanceId.getInstance().getToken();
        Log.d("Not", "Toke ["+tkn+"]");




    }
}

File: thingsalarm2/src/main/java/com/example/thingsalarm/FireIDService.kt
package com.example.thingsalarm

import android.util.Log

import com.google.firebase.iid.FirebaseInstanceId
import com.google.firebase.iid.FirebaseInstanceIdService


class FireIDService : FirebaseInstanceIdService() {

    override fun onTokenRefresh() {
        val tkn = FirebaseInstanceId.getInstance().token
        Log.d("Not", "Toke [$tkn]")


    }

    companion object {

        private val TAG = "MyFirebaseIIDService"
    }
}
